src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.scss

```diff
@@ -63,8 +63,9 @@
   grid-area: moreInformation;
   display: grid;
   grid-template-columns: 190px auto;
-  grid-column-gap: 32px;
+  grid-column-gap: 20px;
```

Comment:
The CSS standard has changed the `grid-column-gap` and `grid-row-gap` properties to just `column-gap` and `row-gap` ([link](https://www.w3.org/TR/css-grid-1/#gutters)).